_posts/2017-04-10-snowplow-javascript-tracker-2.8.0-released.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: post
+title: "Snowplow JavaScript Tracker 2.8.0 released"
+title-short: Snowplow JavaScript Tracker 2.8.0
+tags: [snowplow, javascript, parrable, optimizely, optimizelyx, privacy, optout]
+author: Ben
+category: Releases
+---
+
+We are pleased to announce a new release of the [Snowplow JavaScript Tracker][js-tracker]. [Version 2.8.0][2.8.0-tag] gives you much more flexibility flexibility and control in the area of in-browser user privacy, as well as adding new integrations for Parrable and OptimizelyX.
+
+Read on below the fold for:
+
+1. [State storage strategy](
+/blog/2017/04/10/snowplow-javascript-tracker-2.8.0-released/#storage-strategy)
+2. [Opt-out cookie](
+/blog/2017/04/10/snowplow-javascript-tracker-2.8.0-released/#opt-out)
+3. [Better form tracking for passwords](
+/blog/2017/04/10/snowplow-javascript-tracker-2.8.0-release/#passwords)
+4. [New OptimizelyX and Parrable contexts](
+/blog/2017/04/10/snowplow-javascript-tracker-2.8.0-released/#contexts)
+5. [Extracting key data from the tracker](
+/blog/2017/04/10/snowplow-javascript-tracker-2.8.0-released/#get-data)
+6. [Improved page activity handling](
+/blog/2017/04/10/snowplow-javascript-tracker-2.8.0-released/#page-activity)
+7. [Upgrading](
+/blog/2017/04/10/snowplow-javascript-tracker-2.8.0-released/#upgrade)
+8. [Documentation and help](
+/blog/2017/04/10/snowplow-javascript-tracker-2.8.0-released/#doc)
+
+<!--more-->
+
+<h2 id="storage-strategy">1. State storage strategy</h2>
+
+This release introduces the [`stateStorageStrategy`][sss] property to the
+tracker configuration, taking one of three values: `cookie`, `localStorage` or `none`.
+
+The `cookie` value (the default) will store the id and session data in
+cookies, which was the prior behavior if you set [`useCookies`][uc] to true or left it unspecified. The outgoing event buffer (previously controlled using [`useLocalStorage`][uls]) will continue to use local storage.
+
+**BEN IS THIS CORRECT RE local storage for the event buffer ^^?**
```

Comment:
Hey @BenFradet - interesting. I think your option is internally consistent, but it shouldn't be the default - the default should be the current default behavior. So maybe we add a new setting, `cookieAndLocalStorage`? I feel like this section of the blog post would then be clearer with a table of the four settings as rows, and then what gets stored where as columns... Does that make sense?